bigbluebutton-html5/public/locales/en.json

```diff
@@ -174,6 +174,7 @@
     "app.userList.userOptions.clearAllLabel": "Clear all status icons",
     "app.userList.userOptions.clearAllDesc": "Clears all status icons from users",
     "app.userList.userOptions.clearAllReactionsLabel": "Clear all reactions",
+    "app.userList.userOptions.invitationItem": "Breakout room invitation",
```

Comment:
I suggest to improve the ID making it clearer that the invitation is related with breakoutRoom.
E.g: `app.userList.userOptions.breakoutRoomInvitationLabel`.

And looks like all the other buttons has a `Desc` and only this one is missing. Is it expected?

Reply:
Some items have no description, such as `saveUsersName` - Which leads me into thinking it is not mandatory. But in case you think it is helpful, a good practice or beneficial in any way, just let me know and I'll add description to the items that do not have them.